### PR TITLE
update cert-manager Helm chart to 1.15.1

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
The upgrade notes for 1.15 contain nothing relevant for us: https://cert-manager.io/docs/releases/upgrading/upgrading-1.14-1.15 -- so I think this should be a safe upgrade.

**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
update cert-manager Helm chart to 1.15.1
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: charts/cert-manager/test/default.yaml.out [hunk 1]
```diff
@@ -1116,6 +1118,25 @@ spec:
...
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
+# Source: cert-manager/charts/cert-manager/templates/crds.yaml
+# START crd
+---
```
Comment (wurbanski) [inline charts/cert-manager/test/default.yaml.out:1139]: Where does this come from? are there some CRDs we should be installing?

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

File: charts/cert-manager/test/default.yaml.out (hunk 1 above)
Comment (ahmedwaleedmalik) [reply charts/cert-manager/test/default.yaml.out:1139]: @wurbanski Sorry, I saw this comment after approving the PR. @xrstf please take a look.

**Pull request merged**